### PR TITLE
Set aria-hidden on exclude icon

### PR DESCRIPTION
**Pull request by olamothe**
https://coveord.atlassian.net/browse/JSUI-3074

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

File: src/ui/Facet/ValueElementRenderer.ts [hunk 1]
```diff
@@ -62,8 +62,11 @@ export class ValueElementRenderer {
       title: l('ExcludeValueWithResultCount', this.caption, l('ResultCount', this.count, parseInt(this.count, 10))),
       className: 'coveo-facet-value-exclude',
       tabindex: 0,
-      role: 'button'
+      role: 'button',
+      'aria-hidden': 'true'
```
Comment (samisayegh) [inline src/ui/Facet/ValueElementRenderer.ts:66]: Could this PR conflict with the work here? https://github.com/coveo/search-ui/pull/1585

Comment (ThibodeauJF) [reply src/ui/Facet/ValueElementRenderer.ts:66]: Screen Shot 2020-09-01 at 9 20 15 AM
Maybe have aria-hidden on the svg itself?

Comment (btaillon) [reply src/ui/Facet/ValueElementRenderer.ts:66]: JF is right, I believe the `aria-hidden` should be on the SVG itself.

File: src/ui/Facet/ValueElementRenderer.ts [hunk 2]
```diff
@@ -170,6 +173,16 @@ export class ValueElementRenderer {
...
 
+  private toggleAriaHiddenOnFocusBlur(elem: Dom) {
+    elem.on(['focus', 'mouseover'], () => {
+      elem.setAttribute('aria-hidden', 'false');
+    });
+
+    elem.on(['blur', 'mouseout'], () => {
+      elem.setAttribute('aria-hidden', 'true');
+    });
+  }
```
Comment (btaillon) [inline src/ui/Facet/ValueElementRenderer.ts:184]: I don't see a case where the SVG shouldn't be hidden from the accessibility tree. However, the exclude button probably shouldn't ever be hidden so long as it's visible.

**Comment by olamothe** [comment]
Ahh... Okay then, I though they wanted the aria-hidden to be toggled on/off only when the exclude button si actually hovered. 
It's gonna be a simpler change then

**Pull request merged**